src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/PictureBoxActionListTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using System.ComponentModel.Design;
+
+namespace System.Windows.Forms.Design.Tests;
+
+    public sealed class PictureBoxActionListTests
```

Comment:
Indentation looks to be off now that we have changed to file scoped namespace. Please fix the spacing for this class.
```suggestion
public sealed class PictureBoxActionListTests
```